tests/lib/rules/no-empty-static-block.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @fileoverview Tests for no-empty-static-block rule.
+ * @author Sosuke Suzuki
+ */
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require("../../../lib/rules/no-empty-static-block"),
+    { RuleTester } = require("../../../lib/rule-tester");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
```

Comment:
nits: you can use `parserOptions: { ecmaVersion: 2022 }` here, so no longer need to write it in every case. :)

Reply:
Thank you! [849be2a](https://github.com/eslint/eslint/pull/16325/commits/849be2a5b7213d7442d16f785bd8076614bd1d8a)